cdap-docs/_common/common-build.sh

```diff
@@ -66,7 +67,7 @@ TARGET_PATH="${SCRIPT_PATH}/${TARGET}"
 SOURCE_PATH="${SCRIPT_PATH}/${SOURCE}"
 
 if [ "x${2}" == "x" ]; then
-  PROJECT_PATH="${SCRIPT_PATH}/../../"
+  PROJECT_PATH="${SCRIPT_PATH}/../.."
```

Comment:
To make it consistent that paths *don't* end in a "/". ``pwd`` doesn't return a path with a slash, so I think this should follow that.